docs/secure-connections/sauce-connect/system-requirements.md

```diff
@@ -145,373 +145,134 @@ As an important step prior to downloading Sauce Connect Proxy, you or your syste
 
 ## Configuring Your System to Use Sauce Connect
 
-Select a cloud provider from the tables below to view the recommended system requirements:
+Select a cloud provider from the tables below to view the minimum and recommended system requirements:
+
+:::note These are general recommendations; actual system requirements will depend on the number of parallel tests, and amount of data transferred with each test.
```

Comment:
```suggestion
:::note These are general recommendations; actual system requirements will depend on the number of parallel tests, and amount of data transferred with each test.
:::
```

Reply:
@aliciakrebs @amckenzie132 ~ The body of the note should appear on a second line. Otherwise, the note will display as in the screenshot below:

<img width="805" alt="Screenshot 2023-01-13 at 4 52 46 PM" src="https://user-images.githubusercontent.com/15352964/212426032-7d639a24-62fb-45d0-8d49-13378acabf90.png">